src/appkit.cs

```diff
@@ -13153,6 +13201,18 @@ partial interface NSScreen {
 		[Mac (10,11)]
 		[Export ("maximumExtendedDynamicRangeColorComponentValue")]
 		nfloat MaximumExtendedDynamicRangeColorComponentValue { get; }
+
+		[Mac (10, 15)]
+		[Export ("maximumPotentialExtendedDynamicRangeColorComponentValue")]
+		nfloat MaximumPotentialExtendedDynamicRangeColorComponentValue { get; }
+
+		[Mac (10, 15)]
+		[Export ("maximumReferenceExtendedDynamicRangeColorComponentValue")]
+		nfloat MaximumReferenceExtendedDynamicRangeColorComponentValue { get; }
+
+		[Mac (10, 15)]
+		[Export ("localizedName", ArgumentSemantic.Copy)]
+		string LocalizedName { get; }
```

Comment:
`localizedName` is an optional in swift according to the [doc](https://developer.apple.com/documentation/appkit/nsrunningapplication/1526751-localizedname). Shouldn't yhis have a `NullAllowed` attribute?